src/lib/updates/index.js

```diff
@@ -5,12 +5,12 @@ import Config from '../../config/config'
 import { fireEvent } from '../analytics/analytics'
 import logger from '../logger/pino-logger'
 
-import profileFix from './profileFix'
-import uploadAvatars from './avatar'
-import updateFeedEvents from './updateFeedEvents'
+import migrateProfiles from './migrations/profiles'
+import migrateAvatars from './migrations/avatars'
+import migrateFeed from './migrations/feed'
```

Comment:
really unnecessary refactoring

Reply:
yes but looks fine and is more readable then 'profileFix2' etc